app/src/main/java/de/westnordost/streetcomplete/quests/air_pump/AddAirCompressor.kt

```diff
@@ -16,13 +16,12 @@ import de.westnordost.streetcomplete.util.ktx.toYesNo
 class AddAirCompressor : OsmFilterQuestType<Boolean>() {
 
     override val elementFilter = """
-       nodes, ways with
-       amenity = fuel
-       and (
-           !compressed_air
-           or compressed_air older today -6 years
-       )
-       and access !~ private|no
```

Comment:
Is there a reason why `and access !~ private|no` was deleted here? Admittedly there are not that many of them but they do [seem to exist](https://overpass-turbo.eu/s/1UdT), and leaving the check in should not be harmful?

Reply:
Because this is not about `amenity=air_compressor` but `amenity=fuel`.

Specifying a fuel station as private is a bit of a troll tag.